### PR TITLE
Feat add alert component

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Alert Component**
> Basic Alerts
> Alert Actions

alert

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.avatar {
+    height: var(--sp-4);
+    width: var(--sp-4);
+    border-radius: 50%;
```
Comment (devananth) [inline components/avatar/avatar.css]: Even for `border-radius` , you can have classes like `.rounded` or have a variable. You will using it in badges , images and in many places.

**Review by devananth — CHANGES_REQUESTED** [review]
Clean Code ! Liked the way you used `space` variables.

File: components/avatar/avatar.css (hunk 1 above)
Comment (nerdyspook) [reply components/avatar/avatar.css]: Thanks, I will do that.

**Review by devananth — APPROVED** [review]
You can merge it

**Review by Kavita539 — APPROVED** [review]

**Comment by Kavita539** [comment]
Looks Good!

**Pull request merged**